Requirements for SSE.md

```diff
@@ -98,22 +98,21 @@ Risks of installing third-party add-ons is difficult for Home Assistant to contr
 
 ### Use Case 5
 
-#### TODO: Insert Use Case Name
+#### Tablet Interface Use Case
 
-TODO: Insert use-case info
+Charlie wants to be able to control his various IoT devices thorugh Home Assistant in the living room using a spare tablet. Home Assistant offers a customizable dashboards built in called [LoveLace](https://www.home-assistant.io/lovelace/), which allows for Charlie to set up a simple dashboard for his tablet, to control his various IoT devices with ease, such as his Amazon Alexa, Thermostat, and lights. Once installed Charlie is prompted for his login information to his account, Charlie being the only person set up in his Home Assistant, provides the login information that was orginially created during Home Assistants initaliazation, the Owner Account. Later that night Charlie invites his sister and her children over dinner, excited about his recent endevour, he shows his guests just how much control Home Assistant allows from his tablet.
```

Comment:
Home Assistant offers different users to be created to the system, allowing for a read-only configuration
this sentence is weirdly worded try:
"Home Assistant gives administrator users the ability to create standard user accounts..."
also 
remove I wish and say "Home Assistant should...."

Reply:
That has been changed!